Makefile

```diff
@@ -1,21 +1,22 @@
 # Must have `sentry-cli` installed globally
 # Following variable must be passed in
 #  SENTRY_AUTH_TOKEN
-
-SENTRY_ORG=testorg-az
-SENTRY_PROJECT=java-springboot-logback
-VERSION=`sentry-cli releases propose-version`
+# export JAVA_HOME=/usr/bin/java
+SENTRY_ORG=will-captel
+SENTRY_PROJECT=java-1
+SENTRY_RELEASE=`sentry-cli releases propose-version`
```

Comment:
java documentation for the env vars calls it SENTRY_RELEASE, and we started calling it this in other demo apps, as the value passed becomes the 'Release', and not a 'version' in Sentry